frontend/src/modules/_shared/LayerFramework/settings/implementations/DrilledWellboresSetting.tsx

```diff
@@ -46,26 +39,26 @@ export class DrilledWellboresSetting implements Setting<ValueType> {
         return matchingValues;
     }
 
-    makeComponent(): (props: SettingComponentProps<ValueType>) => React.ReactNode {
-        return function DrilledWellbores(props: SettingComponentProps<ValueType>) {
+    makeComponent(): (props: SettingComponentProps<ValueType, SettingCategory.MULTI_OPTION>) => React.ReactNode {
+        return function DrilledWellbores(props: SettingComponentProps<ValueType, SettingCategory.MULTI_OPTION>) {
+            const availableValues = React.useMemo(() => props.availableValues ?? [], [props.availableValues]);
```

Comment:
Is this needed? As `availableValues` only will be reference stable as long as `props.availableValues` is reference stable, i.e. just use `props.availableValues`?